01-js-intro-and-jquery/Topic 1:JavaScriptIntro and JQery part 2/js/script.js

```diff
@@ -0,0 +1,29 @@
+$(document).on('ready', function() {
+    $section = $('.section');
+    $.ajax ({
+        url: 'https://api.spotify.com/v1/search?q=Rolling&type=album',
+        success: function(response) {
+            var albums = response.albums.items;
+            var albumCover = '';
```

Comment:
Why you need to initialize it if you are setting this up for each album? It don't seems needed for me. Not need to change for this exercise.